src/main/java/org/reaktivity/nukleus/tls/internal/stream/ClientStreamFactory.java

```diff
@@ -471,6 +476,14 @@ private void handleReset(
             doReset(applicationThrottle, applicationId);
             tlsEngine.closeOutbound();
         }
+
+        private void doResetApplicationStream()
+        {
+            if (applicationWindowBytes == -1)
+            {
+                doReset(applicationThrottle, applicationId);
+            }
+        }
```

Comment:
Rename `doResetApplicationStream` to `handleNetworkReplyDone`.

Reply:
done